src/extension.ts

```diff
@@ -111,14 +120,15 @@ export function activate(context: ExtensionContext) {
         }
       }
       const x = JSON.stringify(defaultValue);
-      settings = JSON.parse(x);
+      xml = JSON.parse(x);
```

Comment:
you're missing the `xml = { "xml" : xml};` bit here. Anyways line should be removed because of previous comment